constantine/eth_verkle_ipa/eth_verkle_constants.nim

```diff
@@ -45,7 +45,7 @@ type VerkleMultiproofSerialized* = array[576, byte]
 
 type 
   PrecomputedWeights* = object
-    barycentricWeights*: array[510,Fr[Banderwagon]]
+    barycentricWeights*: array[512,Fr[Banderwagon]]
```

Comment:
I thought we were skipping 1/k and -1/k with k = 0?

Reply:
Barycentric weights store stores the weights $A_i$ from [-k,  0] and then from [k, 0], whereas inverted domain stores the k inverses. Hence, barycentric weight array is 256 * 2 long, and inverted domain is 510.